### DISCUSSION TITLE
Rc2 - no choice for update retroarch and onion only

### DISCUSSION THREAD

**Discussion by ac3nl** [OnionUI/Onion#715] [Q&A]
### Checklist

- [X] I am reporting a bug in the Onion OS (Note that Onion devs can’t fix inherited issues with RA Cores)
- [X] I have reviewed the [Onion Wiki](https://github.com/OnionUI/Onion/wiki) and did not find the answer
- [X] I have searched the existing (open and closed) issues to ensure I do not log a duplicate

### Onions OS Version

4.1 rc2

### Miyoo Firmware Version

20220419

### Provide a clear and concise description of the issue

When installing rc2 , there is no choice for the update only onion and retroRch mode

i am stuck in a loop where preparing update is on the screen and nothing happens
Removing battery is the only option now

### Steps to Reproduce

Copied all the files in the root dir
Swich on

### Provide any additional context or information that may help us investigate your issue

I have some custom emu’s installed (copy’s of existing , to split msx, neogeopocket)

**Comment by Aemiii91** [comment]
This is not a bug.

`Preparing update...` can take a long time depending on the speed of your SD card (it cleans a lot of files).

The installer won't touch your custom emus (as long as their folders are named something different than the Onion emus).

**Comment by ac3nl** [comment]
Why wasn”t the menu there anymore?
